DuckDuckGoTests/DailyPixelTests.swift

```diff
@@ -349,19 +354,14 @@ final class DailyPixelTests: XCTestCase {
         wait(for: [expectation], timeout: 3.0)
     }
 
-    func testThatDailyPixelWithCountWillAppendDToPixelNameForDaily() {
+    func testThatDailyPixelWithLegacyPixelSuffixAndCountWillAppendDAndC() {
         let expectation = XCTestExpectation()
 
-        stub { request in
-            request.url?.absoluteString.contains(Pixel.Event.forgetAllPressedBrowsing.name + "_d") == true
-        } response: { _ in
-            return HTTPStubsResponse(data: Data(), statusCode: 200, headers: nil)
-        }
```

Comment:
The stubs in this test file weren't actually being used, since each test is using a mock pixel sender, so these tests were effectively not really testing anything. I've changed them to instead look at the pixel names sent to the mock.